modules/database/base/utils.py

```
@@ -0,0 +1,59 @@
+    for column in columns:
+        if column not in ignore_columns:
+            pairs[column.name] = sql_type_to_human_type_string(column.type)
+    return pairs
+
+
+def sql_type_to_human_type_string(sql_type) -> str:
+    if not hasattr(sql_type, "python_type"):
+        return ""
+
+    python_type = sql_type.python_type
+    if python_type == str:
```

Ruff (0.8.0)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)